docs/user-guide/kubernetescurator.rst

```diff
@@ -385,3 +385,85 @@ To download data from your PVC, you can use the ``nemo-workspace-busybox`` Pod c
 
     # Copy directory in PVC /nemo-workspace/fizzbuzz to local file-system at $LOCAL_WORKSPACE/fizzbuzz
     kubectl exec nemo-workspace-busybox -- tar cf - /nemo-workspace/fizzbuzz | tar xf - -C $LOCAL_WORKSPACE
+
+
+Running existing Modules using Kubernetes Python client:
```

Comment:
1. The formatting on this title is incorrect. Delete the blank line between the text and the `------`
1. The title is vague and implies this does the same thing as the section `(Opt #1) Running Existing Module`. Could you please refine the title to make it clear how it's different than the other section.
1. Prefix the title with `(Opt #3)` and move it after the `(Opt #2)` section. Or, it might make more sense to have this be `(Opt #2)` and move the old `(Opt #2)` to `(Opt #3)` depending on the purpose of this section.